go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/containernetworking/cni v1.1.1
 	github.com/coreos/go-iptables v0.6.0
 	github.com/digitalocean/go-qemu v0.0.0-20220826173844-d5f5e3ceed89
-	github.com/docker/cli v25.0.3+incompatible
+	github.com/docker/cli v24.0.9+incompatible
```

Comment:
question: Out of curiosity, why was this downgraded?

Reply:
Good catch, by mistake!